sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/TelemetryPartA.cs

```diff
@@ -49,6 +52,33 @@ internal static TelemetryItem GetTelemetryItem(Activity activity, string instrum
             return telemetryItem;
         }
 
+        internal static TelemetryItem GetTelemetryItem(LogRecord logRecord, string instrumentationKey)
+        {
+            var name = PartA_Name_Mapping[TelemetryType.Message];
+            var time = logRecord.Timestamp.ToUniversalTime().ToString(CultureInfo.InvariantCulture);
+
+            TelemetryItem telemetryItem = new TelemetryItem(name, time)
+            {
+                InstrumentationKey = instrumentationKey
+            };
+
+            // TODO: I WAS TOLD THIS MIGHT BE CHANGING. IGNORING FOR NOW.
+            //InitRoleInfo(activity);
+            //telemetryItem.Tags[ContextTagKeys.AiCloudRole.ToString()] = RoleName;
+            //telemetryItem.Tags[ContextTagKeys.AiCloudRoleInstance.ToString()] = RoleInstance;
+            telemetryItem.Tags[ContextTagKeys.AiOperationId.ToString()] = logRecord.TraceId.ToHexString();
+
+            // TODO: HOW TO HANDLE PARENT CONDITION?
+            //if (activity.Parent != null)
```

Comment:
you shouldn't have to worry about Acitivty or parent in the Exporter. LogRecord should contain Traceid,spanid, if there was a valid parent.